### ISSUE TITLE
./configure fails with Xcode 6.3: "thread-local storage is not supported for the current target"

### ISSUE THREAD

**Issue by dusek** [textmate/bugs#14]
```
FAILED: xcrun clang++ -include /Users/xxx/build/TextMate/Shared/PCH/prelude.cc -c -pipe -fPIC -gdwarf-2 -m64 -mmacosx-version-min=10.7 -funsigned-char -D'NULL_STR="\uFFFF"' -DREST_API='"https://api.textmate.org"' -Wall -Wwrite-strings -Wformat -Winit-self -Wmissing-include-dirs -Wno-parentheses -Wno-sign-compare -Wno-switch -IShared/include -fcolor-diagnostics -DNDEBUG -Os -fvisibility=hidden -std=c++1y -stdlib=libc++ -I"/usr/local/include"  -o /Users/xxx/build/TextMate/Frameworks/crash/src/info.o -MMD -MF /Users/xxx/build/TextMate/Frameworks/crash/src/info.o.d -I/Users/xxx/build/TextMate/include Frameworks/crash/src/info.cc
Frameworks/crash/src/info.cc:61:3: error: thread-local storage is not supported for the current target
                thread_local stack_t stack;
                ^
```

See also [Apple Dev Forums thread](https://devforums.apple.com/message/1101679#1101679)
I seem to be [not the only one](https://root.cern.ch/phpBB3/viewtopic.php?t=19538&p=83755#p83586) having problems with `thread_local` after upgrading to Xcode 6.3

Output:

``` bash
$ clang --version
Apple LLVM version 6.1.0 (clang-602.0.49) (based on LLVM 3.6.0svn)
Target: x86_64-apple-darwin14.3.0
Thread model: posix
$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.10.3
BuildVersion:   14D131
```

**Comment by jacob-carlborg** [comment]
I have the same problem: http://comments.gmane.org/gmane.editors.textmate.general/38535

**Comment by dusek** [comment]
Discussed on [TextMate mailing list](http://lists.macromates.com/textmate/2015-April/038602.html).

**Comment by sorbits** [comment]
Should be fixed by textmate/textmate@172ce9d4282e408fe60b699c432390b9f6e3f74a

**Comment by jacob-carlborg** [comment]
Awesome, thanks.
